docs/quorum-queues/index.md

```diff
@@ -778,10 +778,13 @@ of returning the message near the head of the queue.
 The internal implementation of quorum queues converts the queue name
 into an Erlang atom. If queues with arbitrary names are continuously
 created and deleted it _may_ threaten the long term stability of the
-RabbitMQ system (if the size of the atom table reaches the maximum limit,
-about 1M by default). It is not recommended to use quorum queues in this manner
-at this point.
+RabbitMQ system if the size of the atom table reaches the maximum limit, 
+5 million by RabbitMQ default (about 1 million by erlang default). 
+It is not recommended to use quorum queues in this manner at this point.  
 
+Howver, if needed, and at the cost of potentially increased memory usage, 
```

Comment:
This does not belong to the Quorum Queues guide. Please add a new section to the Runtime guide and link to it.